ui/src/app/views/workspace-nav-bar/component.css

```diff
@@ -0,0 +1,69 @@
+::ng-deep .btn.btn-primary.btn-top {
```

Comment:
What are the "ng-deeps" for ? I read the summary of this feature but not sure about the application to this code in particular.

nit: add comment?

Reply:
Yup, just for your info here, its to plumb styles down into the clarity components, because for button overflow in particular, I couldn't actually style the dropdown button without it.